src/meta/src/hummock/hummock_manager.rs

```diff
@@ -178,7 +178,8 @@ macro_rules! start_measure_real_process_timer {
 #[derive(Default)]
 struct Versioning {
     current_version_id: CurrentHummockVersionId,
-    hummock_versions: BTreeMap<HummockVersionId, HummockVersion>,
+    current_version: Option<HummockVersion>,
```

Comment:
Why it need to be `Option`? 
Could we have an initial version?

Reply:
> Why it need to be `Option`?
> Could we have an initial version?

initial version is generated when `load_mem_store` is called, which is too late

Reply:
> Why it need to be `Option`? Could we have an initial version?

fixed